core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -2699,6 +2699,35 @@ private void checkProjectCorrelateTransposeRuleSemiOrAntiCorrelate(JoinRelType t
         .check();
   }
 
+  /** Tests a variant of {@link FilterProjectTransposeRule}
+   * that pushes a Filter that contains a correlating variable. */
```

Comment:
Is `x1` a correlating variable?

Reply:
I think not. @kasakrisz, can you correct the javadoc.

Reply:
I copy-pasted an existing test hence the misleading comment. Fixed.